core/src/main/scala/org/bitcoins/core/wallet/utxo/InputInfo.scala

```diff
@@ -223,34 +243,40 @@ object RawInputInfo {
               "ConditionalPath must be specified for P2PKWithTimeout")
           case Some(beforeTimeout) =>
             P2PKWithTimeoutInputInfo(outPoint,
+                                     prevTransaction,
                                      amount,
                                      p2pkWithTimeout,
                                      beforeTimeout)
         }
       case multiSig: MultiSignatureScriptPubKey =>
-        MultiSignatureInputInfo(outPoint, amount, multiSig)
+        MultiSignatureInputInfo(outPoint, prevTransaction, amount, multiSig)
       case conditional: ConditionalScriptPubKey =>
         ConditionalInputInfo(outPoint,
+                             prevTransaction,
                              amount,
                              conditional,
                              conditionalPath,
                              hashPreImages)
       case lockTime: LockTimeScriptPubKey =>
         LockTimeInputInfo(outPoint,
+                          prevTransaction,
                           amount,
                           lockTime,
                           conditionalPath,
                           hashPreImages)
       case EmptyScriptPubKey =>
-        EmptyInputInfo(outPoint, amount)
+        EmptyInputInfo(outPoint, prevTransaction, amount)
       case _: NonStandardScriptPubKey | _: WitnessCommitment =>
         throw new UnsupportedOperationException(
           s"Currently unsupported ScriptPubKey $scriptPubKey")
     }
   }
 }
 
-case class EmptyInputInfo(outPoint: TransactionOutPoint, amount: CurrencyUnit)
+case class EmptyInputInfo(
+    outPoint: TransactionOutPoint,
+    prevTransaction: Transaction,
+    amount: CurrencyUnit)
```

Comment:
Since `EmptyInputInfo` is already only for tests you should just override `prevTransaction` to be `EmptyTransaction` in the body rather than taking in a param that could conflict with `outPoint`

Reply:
Also, this becomes somewhat irrelevant once you look at my comment above so :man_shrugging: